### ISSUE TITLE
Annealing the learning rate:

### ISSUE THREAD

**Issue by AidanRocke** [AidanRocke/variational_empowerment#1]
1. Decreasing the learning rate corresponds to increasing confidence over time. 
2. Several approaches to try which are recommended [here](http://cs231n.github.io/neural-networks-3/#anneal): 

a. Step decay: Reduce the learning rate by some factor every few epochs. Typical values might be reducing the learning rate by a half every 5 epochs, or by 0.1 every 20 epochs. These numbers depend heavily on the type of problem and the model. One heuristic you may see in practice is to watch the validation error while training with a fixed learning rate, and reduce the learning rate by a constant (e.g. 0.5) whenever the validation error stops improving.
b. Exponential decay. has the mathematical form α=α0e−kt, where α0,k are hyperparameters and t is the iteration number (but you can also use units of epochs).
c. 1/t decay has the mathematical form α=α0/(1+kt) where a0,k are hyperparameters and t is the iteration number.
